roles/keycloak/defaults/main.yml

```diff
@@ -19,7 +19,7 @@ keycloak_config_override_template: ''
 keycloak_config_path_to_properties: "{{ keycloak_jboss_home }}/standalone/configuration/profile.properties"
 keycloak_service_user: keycloak
 keycloak_service_group: keycloak
-keycloak_service_pidfile: "/run/keycloak.pid"
+keycloak_service_pidfile: "/run/keycloak/keycloak.pid"
```

Comment:
changing this on existing deployments requires to add daemon_reload in the restart handler https://github.com/ansible-middleware/keycloak/blob/345c50fb850d23c1cde5f183e7b6456b6eae4842/roles/keycloak/tasks/systemd.yml#L11

Reply:
Also the unit file template contains that parameter so this should trigger the reload
https://github.com/ansible-middleware/keycloak/blob/345c50fb850d23c1cde5f183e7b6456b6eae4842/roles/keycloak/tasks/systemd.yml#L30C2-L30C2